src/cljs/vanilla/widgets/make_chart.cljs

```diff
@@ -228,4 +253,102 @@
    :conversions       {:default             "",
                        :data-format/from-to ""}}
 
+  ;;;;;;;;;;;;ALL-CONFIGS;;;;;;;;;;;;; for future reference
```

Comment:
we need to decide how long to carry this with us

Reply:
Fair!  Can always purge it, or move it to a documentation page or something.  I think of all the comment blocks, i reference that all-config most often when changing things.  Nice to see the final final data struct thats being sent to highcharts.  Im fine with removing, moving it, or just tossing it.  can always print it out again if we need to debug that deep in the config.

Reply:
I’m cool with keeping it. Let’s comment to justify why we have it and what another dev should do with it.